src/System.Security.Cryptography.Algorithms/src/Resources/Strings.resx

```diff
@@ -214,6 +214,9 @@
   <data name="Cryptography_OpenInvalidHandle" xml:space="preserve">
     <value>Cannot open an invalid handle.</value>
   </data>
+  <data name="Cryptography_Padding_DecDataTooBig" xml:space="preserve">
+    <value>The data to be decrypted exceeds the maximum for this modulus of {0} bytes.</value>
```

Comment:
This is technically correct I suppose, but I wonder if it's really actionable by the developer. Would the dev know that modulus is generally a synonym for "key size", and would they also know how that relates back to maximum message length? Seems like we may want to reword this to simply "The data to be decrypted exceeds the maximum length for this key."

Reply:
It's not quite what I would have done if doing it fresh, but it's what RSACryptoServiceProvider has been doing for years.  So this is just about making the other implementations behave accordingly.

https://referencesource.microsoft.com/#mscorlib/system/security/cryptography/rsacryptoserviceprovider.cs,418